sql/catalyst/pom.xml

```diff
@@ -60,6 +60,11 @@
       <artifactId>scalacheck_${scala.binary.version}</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
actually i think the proper way is to define the version in the root pom, and then here just pull janino in.